app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Admin::OrdersController < Admin::BaseController
+  def index
```

Comment:
do we need an index? or is this what we are holding off for with the user story 31?